manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -240,6 +240,12 @@ data:
     #
     # owners: (https://github.com/kubevirt)
     #
+    # cnv_abnormal represents the reason why the operator might have an issue
+    # and includes the node, namespace, container, reason labels.
```

Comment:
Why include the node name? Is it meaningful?

Reply:
the `node` label wasn't mentioned in https://issues.redhat.com/browse/MON-3763

Reply:
Yes,, we need to node label for sure.

Reply:
but the namespace needs to be removed